R/query_dataset.R

```diff
@@ -5,49 +5,92 @@
 #' codes to the relevant flags to filter on time period, geographic level, location,
 #' and data set specific filters. If none of the above are set in the function call,
 #' then the entire data set will be retrieved. The data set id and specific indicators
-#' of interest must be supplied explictly using the dataset_id and indicators params.
+#' of interest must be supplied explicitly using the dataset_id and indicators params.
 #'
 #' @inheritParams api_url
+#' @inheritParams post_dataset
```

Comment:
Should we have the option to specify rows that only have the exact combination of filter options too? Currently can't see how to do that

Reply:
I'd not exampled that bit properly yet, I've added an example now and some explanation in then description